File: Java-Core/src/main/java/presto/python/Python2NativeCall.java
package presto.python;

import presto.utils.CodeWriter;

public class Python2NativeCall extends PythonNativeCall {

	public Python2NativeCall(PythonStatement statement) {
		super(statement);
	}

	@Override
	public void toDialect(CodeWriter writer) {
		writer.append("Python2: ");
		super.toDialect(writer);
	}
	
	
}

File: Java-CodeServer/src/main/resources/js/prompto/python/Python2NativeCall.js
var PythonNativeCall = require("./PythonNativeCall").PythonNativeCall;

function Python2NativeCall(statement, module) {
	PythonNativeCall.call(this, statement, module);
	return this;
}

Python2NativeCall.prototype = Object.create(PythonNativeCall.prototype);
Python2NativeCall.prototype.constructor = Python2NativeCall;

Python2NativeCall.prototype.toDialect = function(writer) {
    writer.append("Python2: ");
    PythonNativeCall.prototype.toDialect.call(this, writer);
};

exports.Python2NativeCall = Python2NativeCall;